Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
```

Comment:
I typically add the following line here:
```
ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
```
This will make it so you don't have to specify `--provider=virtualbox` to the `vagrant up` command line.

I find myself typing this command frequently enough that I don't like having to add the extra flag.